MapboxCoreNavigation/MBNavigator.swift

```diff
@@ -14,6 +14,7 @@ extension MBFixLocation {
     }
 }
 
+/// :nodoc:
 extension MBRouteState: CustomStringConvertible {
     public var description: String {
```

Comment:
If possible, make this property `private`, because we don’t want MBRouteState to be part of the public API.

Reply:
This property is required to be public because this is requirement of public protocol `CustomStringConvertible` it implements. Compiler does not allow this.